tycho-extras/tycho-pomless/src/main/java/org/eclipse/tycho/pomless/TychoBundleMapping.java

```diff
@@ -49,7 +49,7 @@ public class TychoBundleMapping extends AbstractTychoMapping {
     @Override
     protected boolean isValidLocation(String location) {
         File polyglotArtifactFile = new File(location);
-        if (polyglotArtifactFile.isDirectory() && polyglotArtifactFile.getName().equals(META_INF_DIRECTORY)) {
+        if (polyglotArtifactFile.getName().equals(META_INF_DIRECTORY) && polyglotArtifactFile.isDirectory()) {
             return new File(polyglotArtifactFile, MANIFEST_MF).exists();
```

Comment:
Maybe `.exists() -> isFile() `to be consitent to your other changes?

Reply:
Yes.
But actually this could be simplified to:
`return new File(location, META_INF_DIRECTORY + "/" + MANIFEST_MF).isFile();`
Which is also more performant because the File-System is only queried once.
To avoid the hard-coded line-delimiter, we can use this:
`return Files.isRegularFile(Path.of(location, META_INF_DIRECTORY, MANIFEST_MF));`